pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -310,6 +311,10 @@ def parallel_devices(self) -> List[Union[torch.device, int]]:
     def root_gpu(self) -> Optional[int]:
         return self.accelerator.root_device.index if not isinstance(self.accelerator, TPUAccelerator) else None
 
+    @property
+    def is_training_type_in_plugins(self) -> bool:
+        return any(isinstance(plug, str) and plug in TrainingTypePluginsRegistry for plug in self.plugins)
+
```

Comment:
n00b question: could you explain why this iterates over all plugins in the connector? from the naming i'd assume this meant to check if a single plugin instance is contained in the registry

Reply:
Yup

Reply:
Yup!